docs/source/user_guide/timeseries.ipynb

```diff
@@ -540,6 +540,7 @@
     "    y_train,\n",
     "    problem_type=\"time series regression\",\n",
     "    max_batches=1,\n",
+    "    automl_algorithm=\"iterative\",\n",
```

Comment:
did you add this for `allowed_model_families` to work?

Reply:
Yep, previously it only just ran the first batch which didn't get great results